src/components/clusters/ClusterDetailsMultiRegion/components/ClusterDetailsTop/components/RecommendedOperatorsAlert/ProductCardView.tsx

```diff
@@ -16,6 +16,8 @@ type ProductCardViewProps = {
     name: string;
     link: string;
   };
+  // Optional prop to control visibility
+  isHidden?: boolean;
```

Comment:
Nitpicking, we don't need this types of comments. This prop is self explanatory, no need to explain what `isHidden` mean.